woocommerce/woocommerce-ios
# [Store onboarding] Retry launch store from error alert

Part of : #9122 


## Description


Adds a "Try again" button to the Unexpected error alert displayed while launching a store.

## Testing instructions

Prerequisite: A new WPCOM site (not launched yet)

We have to simulate an unexpected error to test this.

You can mock `https://public-api.wordpress.com/wpcom/v2/sites/*/launch` request with the below invalid token response
```
{"code":"oauth2_invalid_token","message":"The OAuth2 token is invalid.","data":{"status":401}}
```
- Login into the app using WPCOM site
- Tap on the "Launch your store" store onboarding task
- Tap on "Publish My Store" button
- Once you have mocked the response you should see an "Unexpected error" alert
- Tap on "Try again" to try launching the store again
- You can stop mocking the response now to verify that launching from the error alert works.

## Screenshots

https://user-images.githubusercontent.com/524475/227426934-dd1a5eb7-34ed-41f2-b26d-67298bfa202c.mp4

---
